docs/setup/independent/high-availability.md

```diff
@@ -117,7 +117,8 @@ For **Option 2**: you can skip to the next step. Any reference to `etcd0`, `etcd
    EOF
    ```
 
-   Ensure that the `names` section in `ca-csr.json` matches your own company or personal address, or that you use a suitable default.
+   If desired, modify `ca-csr.json` to add a `names` section. This is not strictly required.
```

Comment:
Replace with:
```
**Optional:** You can modify `ca-csr.json` to add a section for `names`.
{: .note}
```

Reply:
Good call - that's both more clear and more consistent. Do we want to keep the link our to the cfssl wiki or remove it? Keeping it would look like this:

```
   **Optional:** You can modify `ca-csr.json` to add a section for `names`.
   See [the CFSSL wiki](https://github.com/cloudflare/cfssl/wiki/Creating-a-new-CSR) for an example.
   {: .note}
```

Also happy to remove it. I don't really have an opinion.

Reply:
Let's keep it! 👍

Reply:
👍 updated.